src/bridge/DelayBuffer.sol

```diff
@@ -130,6 +130,7 @@ library DelayBuffer {
         uint64 blockNumber,
         uint64 timestamp
     ) internal {
+        // @review we can also prevent this by limiting the value you can set as threshold
```

Comment:
the thought is thresholdBlocks = type(uint64).max and thresholdSeconds = type(uint64).max effectively disables the buffer, so governance could turn off the buffer feature without upgrading the contract.

_setBufferConfig calls updateSyncValidity so saturation at type(unit64).max is handled